_sass/partials/_custom-dropdown.scss

```diff
@@ -0,0 +1,70 @@
+/* Custom Dropdown CSS */
+
+
+// Dropdown arrow/caret
+.caret {
+  display: inline-block;
+  width: 0;
+  height: 0;
+  margin-left: 2px;
+  vertical-align: middle;
+  border-top:   $caret-width-base solid;
+  border-right: $caret-width-base solid transparent;
+  border-left:  $caret-width-base solid transparent;
+}
+
+
+
+//CSS3 media queries
+//larger than 767px
+@media (min-width: 767px) {
+	.dropdown {
+    position: relative;
+    display: inline-block;
+	}
+
+	.dropdown > li {
+	    padding-bottom: 20px;
+	    cursor: pointer;
+	}
+
+	/* Dropdown Content (Hidden by Default) */
+	.dropdown-content {
+	    display: none;
+	    position: absolute;
+	    background-color: #fffc;
```

Comment:
Here is the problem. The color code is not correct. Chrome doesnt identify it. Always use full 6 digit hex codes for colors.

Reply:
yeah.. noticed it when I open in chrome.
I will fix this.